code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -52,7 +52,7 @@
 
 /obj/structure/bed/chair/verb/rotate()
 	set name = "Rotate Chair"
-	set category = "Object"
+	set category = null //So it's only accessible from the right click menu
```

Comment:
Set hidden = 1 makes it not appear in the right click menu either, and I want it to appear there. I did this because that's how the DM ref says to do that.

The mob verb now appears in the Object panel, as explained in the OP.